vm/chipset_device/src/io/deferred.rs

```diff
@@ -38,7 +39,7 @@ use std::task::ready;
 /// IOs.
 ///
 /// Create with [`defer_read`] or [`defer_write`].
-#[derive(Debug)]
+#[derive(Debug, MeshPayload)]
```

Comment:
Oh that's crazy, hah.

I think I wouldn't do this--I would resolve the thing locally and then send the result back to the proxy.

Ultimately I think we'll want to keep a single channel for remote device responses instead of using one shots, since establishing and tearing down a oneshot across processes adds an extra Unix socket message per request.